### PR TITLE
In product $plugins_xxxx docs

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

Part of https://github.com/PostHog/posthog.com/issues/1396

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)
- [ ] Breaking changes are backwards-compatible. Ensure old/new frontend requests work with new/old backends, and vice versa.

### CODE REVIEW

File: frontend/src/lib/components/PropertyKeyInfo.tsx [hunk 1]
```diff
@@ -229,6 +229,19 @@ export const keyMapping: KeyMappingInterface = {
...
+            description: 'Plugins that ran successfully on an event.',
+        },
+        $plugins_failed: {
+            label: 'Plugins Failed',
+            description: 'Plugins that tried to run on an event but failed.',
+        },
+        $plugins_deferred: {
+            label: 'Plugins Deferred',
+            description:
+                'Plugins that had read-only access to an event for some asynchronous processing (e.g. for exporting it).',
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyKeyInfo.tsx:243]: Maybe mention which of these relate specifically to `processEvent` and which to `onEvent`/`exportEvents` – that's the crucial difference.

Comment (yakkomajuri) [reply frontend/src/lib/components/PropertyKeyInfo.tsx:243]: I guess that's worth it - might confuse non-technical users though (or even technical non-plugin dev users)

File: frontend/src/lib/components/PropertyKeyInfo.tsx [hunk 2]
```diff
@@ -229,6 +229,19 @@ export const keyMapping: KeyMappingInterface = {
             label: 'Capture Failed Request',
             description: '',
         },
+        $plugins_succeeded: {
+            label: 'Plugins Succeeded',
+            description: 'Plugins that ran successfully on an event.',
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyKeyInfo.tsx:234]:
```suggestion
            description: <>Plugins that successfully processed the event, e.g. edited properties (plugin method <code>processEvent<code/>).</>,
```

File: frontend/src/lib/components/PropertyKeyInfo.tsx [hunk 3]
```diff
@@ -229,6 +229,19 @@ export const keyMapping: KeyMappingInterface = {
...
+        },
+        $plugins_failed: {
+            label: 'Plugins Failed',
+            description: 'Plugins that tried to run on an event but failed.',
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyKeyInfo.tsx:238]:
```suggestion
            description: <>Plugins that failed to process the event (plugin method <code>processEvent<code/>).</>,
```

File: frontend/src/lib/components/PropertyKeyInfo.tsx [hunk 4]
```diff
@@ -229,6 +229,19 @@ export const keyMapping: KeyMappingInterface = {
...
+        },
+        $plugins_deferred: {
+            label: 'Plugins Deferred',
+            description:
+                'Plugins that had read-only access to an event for some asynchronous processing (e.g. for exporting it).',
+        },
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyKeyInfo.tsx:244]:
```suggestion
            description:
                <>
                    Plugins to which the event was handed off post-ingestion, e.g. for export (plugin method <code>onEvent<code/>).
                </>,
        },
```

**Pull request merged**